watchtower-plugin/src/net/http.rs

```diff
@@ -154,6 +155,31 @@ pub async fn send_appointment(
     }
 }
 
+/// Handles a ping request to the tower.
+pub async fn ping(server_url: &str) -> Result<(), String> {
```

Comment:
nit: I'd call this `tower_net_addr` for consistency with the rest of the methods

Reply:
Sure!